client/templates/profile/profile.html

```diff
@@ -0,0 +1,83 @@
+<template name="profile">
+<div id="profile">
+  <section class="profile-top">
+    <div class="container">
+      <div class="row">
+        {{#if editMode}}
+          {{#with userInfo}}
+            {{> profileEdit}}
+          {{/with}}
+        {{else}}
+          <div class="col-md-6 profile-top-left">
+            {{#with userInfo}}
+              <div class="col-md-3">
+                <img src="{{#if profile.avatar.image_192 }} {{profile.avatar.image_192}} {{else}} {{profile.avatar.default}} {{/if}}" class="img-circle" alt="profile image"/>
+              </div>
+              <div class="col-md-9">
+                <h3>{{username}}
+                  {{#if isCurrentUserProfile currentUser}}
+                    <a href="#profileEdit" class="editProfile"><i class="fa fa-pencil" aria-hidden="true"></i>{{_ "Edit Profile"}}</a>
+                  {{else}}
+                    {{#if status.online}}
+                      <a><i class="fa fa-circle online-now" aria-hidden="true"></i></a>
+                    {{/if}}
+                  {{/if}}
+                </h3>
+                <p>
+                  <i class="fa fa-map-marker" aria-hidden="true"></i>{{profile.location}}
+                </p>
+                <div class="social-icons">
+                  <a href="https://twitter.com/{{profile.social.twitter}}" target="_blank">
```

Comment:
can you wrap  `optional` elements like location, twitter, etc... with `if` block ? 
so that could only displayed when those element has some value in it.
i.e. `{{#if profile.social.twitter}} <a>...</a> {{/if}}`